node_modules.py

```diff
@@ -415,6 +415,11 @@ def update_checksum(fn):
                     logging.error("failed to create tar %s", url)
                     continue
             else:
+                if args.alternative_dependencies_server is not None:
+                    url_alternative=args.alternative_dependencies_server
+                    if not url_alternative.endswith("/"):
+                        url_alternative=url_alternative+"/"
+                    url = url.replace("https://registry.npmjs.org/", url_alternative)
```

Comment:
while this is the default registry, what happens when the registry is already different? `npm config get registry` Do you think it makes sense here to replace the modified default, ie. when the `registry` is set in configuration?